app/pages/project/affinity/AffinityPage.tsx

```diff
@@ -64,13 +58,26 @@ export async function clientLoader({ params }: LoaderFunctionArgs) {
 
 const colHelper = createColumnHelper<AntiAffinityGroup>()
 
-export const AffinityPageHeader = ({ name = 'Affinity' }: { name?: string }) => (
-  <PageHeader>
-    <PageTitle icon={<Affinity24Icon />}>{name}</PageTitle>
-    {/* TODO: Add a DocsPopover with docLinks.affinity once the doc page exists */}
-  </PageHeader>
-)
-
+export const AffinityPageHeader = ({ name = 'Affinity' }: { name?: string }) => {
+  const { project } = useProjectSelector()
+  return (
+    <PageHeader>
+      <PageTitle icon={<Affinity24Icon />}>{name}</PageTitle>
+      <div className="inline-flex gap-2">
+        {/* TODO: Add a DocsPopover with docLinks.affinity once the doc page exists */}
+        {name !== 'Affinity' && (
+          <MoreActionsMenu label="Anti-affinity group actions">
+            <DropdownMenu.LinkItem
+              to={pb.antiAffinityGroupEdit({ project, antiAffinityGroup: name })}
+            >
+              Edit
+            </DropdownMenu.LinkItem>
+          </MoreActionsMenu>
+        )}
+      </div>
+    </PageHeader>
+  )
+}
```

Comment:
I don't think it's worth sharing this header between the two call sites and having this name logic. I'd just inline the appropriate thing at each all site.